lib/actions.js

```diff
@@ -188,3 +188,13 @@ actions.addNullDataSourceItem = function() {
     value: null
   });
 };
+
+actions.templateFilter = function() {
+  if (this.options.loopbackVersion === '2.x') {
```

Comment:
The action name templateFilter does not make it clear where it's taking the `loopbackVersion` option, it creates coupling that's difficult to spot. I'd prefer to keep this function inside the generator that's using it. I assume you had a reason for moving it to lib/actions, what was it?